core-models/src/main/scala/com/mesosphere/usi/core/models/PodRecord.scala

```diff
@@ -0,0 +1,27 @@
+package com.mesosphere.usi.core.models
+
+import java.time.Instant
+
+/**
+  * Pod record is an immutable event of what happened to all pods.
+  * @param sequenceNr number used for ordering the pod events.
+  * @param podId id of the pod
+  * @param launchedAt when the pod was launched
+  * @param unserviceable if not none, pod is not in a serviceable state
```

Comment:
What is a `serviceable` state?

Reply:
Suggestion: let's leave out `unserviceable` and `UnserviceableDetails` for now. Let's not support (health) checks in the initial interface, and add it later with a fully designed spec.

Questions:
- would readiness also be rendered as `unserviceable`?
- How do we express that a pod was last seen unhealthy and is now unreachable?
- How would we expose the payload of a (health) check?
- How would we expose the combination of a health check **and** a check? (Mesos supports having each of these.)

As a reference, see the [`CheckStatusInfo`](https://github.com/apache/mesos/blob/40210dbcfb218d6ef3f647b047afbdda24133f4d/include/mesos/v1/mesos.proto#L2464), which will have a payload depending on the type of check.

I'd defer this part for later design and decision.